runtime/ops/signal.rs

```diff
@@ -5,24 +5,26 @@ use deno_core::ZeroCopyBuf;
 use std::cell::RefCell;
 use std::rc::Rc;
 
-#[cfg(unix)]
+#[cfg(any(unix, windows))]
 use deno_core::error::bad_resource_id;
-#[cfg(unix)]
+#[cfg(any(unix, windows))]
 use deno_core::AsyncRefCell;
-#[cfg(unix)]
+#[cfg(any(unix, windows))]
 use deno_core::CancelFuture;
-#[cfg(unix)]
+#[cfg(any(unix, windows))]
```

Comment:
Remove `#[cfg(any(unix, windows))]` everywhere -> it is always true (practically speaking).